extensions/keyboard-brightness/src/utils.ts

```diff
@@ -67,8 +67,8 @@ const adjustBrightness = async (
     }
 
     const adjustment = direction === "increase" ? 0.1 : -0.1;
-    const newBrightness = brightness + adjustment;
-
+    let newBrightness = brightness + adjustment;
+    newBrightness = newBrightness < 0 ? 0 : newBrightness > 1 ? 1 : newBrightness;
```

Comment:
```suggestion
    // ensure that brightness is between 0 and 1
    newBrightness = Math.max(0, newBrightness);
    newBrightness = Math.min(1, newBrightness);
```

Can you perhaps change it to this and add a comment because your original line is really hard to read. Thanks 🙏🏻